src/kio/static/base.py

```diff
@@ -0,0 +1,5 @@
+__all__ = ("ApiMessage",)
+
+
+class ApiMessage:
```

Comment:
`ApiMessage` is moved to a separate module to not cause circular dependency errors, it is also not really an interface/protocol but rather used as a nominal marker for top-level API entities.

We should probably consider redesigning this to be more inline with the rest of the introspection features, and not require an import and subclassing.